setup.py

```diff
@@ -43,7 +42,8 @@
     ],
     include_package_data=True,
     install_requires=[
-
+        'django-formtools>=2.0',
```

Comment:
> how would those two libraries get installed locally by running pip?

`pip install wizard_builder`, when run from outside of a `wizard_builder` folder, consults `setup.py` to see if any requirements should be installed first. It looks like this

```bash
$pip install -e ../django-wizard-builder/
Obtaining file:///Users/lynn/apps/django-wizard-builder
Requirement already satisfied: django-formtools>=2.0 in ./.venv/lib/python3.5/site-packages (from django-wizard-builder==0.1.1)
Requirement already satisfied: django-model-utils>=3.0 in ./.venv/lib/python3.5/site-packages (from django-wizard-builder==0.1.1)
Requirement already satisfied: Django>=1.8 in ./.venv/lib/python3.5/site-packages (from django-formtools>=2.0->django-wizard-builder==0.1.1)
Installing collected packages: django-wizard-builder
  Found existing installation: django-wizard-builder 0.1.1
    Uninstalling django-wizard-builder-0.1.1:
      Successfully uninstalled django-wizard-builder-0.1.1
  Running setup.py develop for django-wizard-builder
Successfully installed django-wizard-builder
```